collection-scripts/gather_windows_node_logs

```diff
@@ -4,8 +4,10 @@ WINDOWS_NODE_LOGS=$BASE_COLLECTION_PATH/host_service_logs/windows
 
 # Logfile list
 LOGS=(kube-proxy/kube-proxy.exe.INFO kube-proxy/kube-proxy.exe.ERROR kube-proxy/kube-proxy.exe.WARNING)
-LOGS+=(hybrid-overlay/hybrid-overlay.log kubelet/kubelet.log)
+LOGS+=(hybrid-overlay/hybrid-overlay.log)
+LOGS+=(kubelet/kubelet.log)
 LOGS+=(containerd/containerd.log)
+LOGS+=(wicd/windows-instance-config-daemon.exe.INFO wicd/windows-instance-config-daemon.exe.ERROR wicd/windows-instance-config-daemon.exe.WARNING)
```

Comment:
Should we just collect /var/log?

Reply:
Wouldn't we pick up container logs also if we do that?

Reply:
Okay I'm fine with how it is currently then